### DISCUSSION TITLE
Can't use $ as prompt character

### DISCUSSION THREAD

**Discussion by poperigby** [starship/starship#1841] [Q&A] [answered]
Hey, I'm trying to get `$` to be my prompt character but it keeps throwing this error:
```
[WARN] - (starship::modules::character): Error in module `character`:
 --> 1:3
  |
1 | [$](bold yellow)
  |   ^---
  |
  = expected variable_name
```
I'm guessing it's because it sees the `$` and assumes a variable name will follow. I tried escaping it like this `\$`, but that didn't fix it.

**Comment by ghost** [comment, marked as answer]
You have to double escape it like this:
`[\\$](bold yellow)`
Because the TOML parser interprets `\$` as an escape sequence.

**Reply by poperigby** [reply]
Perfect, thanks!
